### PR TITLE
Add JWT support

### PR DESCRIPTION
**Pull request by jaapromijn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,16 @@ USERBUNDLE_OAUTH_GOOGLE_SCOPE='email profile'
...
 
+## JTW Token support
+
+If the app needs JTW token support, the Lexik JWTi Authentication bundle should be required and suitable configuration should be added.
+
+Intstall the bundle:
+``` bash
+composer require lexik/jwt-authentication-bundle
+```
+
```
Comment (reyostallenberg) [inline README.md]: Double newline

File: src/DependencyInjection/ConnecthollandUserExtension.php [hunk 2]
```diff
@@ -32,13 +32,34 @@ public function load(array $configs, ContainerBuilder $container)
             new FileLocator(__DIR__.'/../Resources/config')
         );
 
+        if (isset($container->getParameter('kernel.bundles')['LexikJWTAuthenticationBundle'])) {
+            $loader->load('lexik_jwt_authentication.yaml');
+        }
+
```
Comment (reyostallenberg) [inline src/DependencyInjection/ConnecthollandUserExtension.php]: Did you run cs fixer?

File: src/DependencyInjection/ConnecthollandUserExtension.php [hunk 3]
```diff
@@ -32,13 +32,34 @@ public function load(array $configs, ContainerBuilder $container)
...
+            $container->prependExtensionConfig('lexik_jwt_authentication', $config);
+        }
+    }
+
+    private function getJwtConfiguration(ContainerBuilder $container): array
+    {
+        $config = [
+            'secret_key'  => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_SECRET_KEY)'), true),
+            'public_key'  => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_PUBLIC_KEY)'), true),
+            'pass_phrase' => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_PASSPHRASE)'), true),
```
Comment (reyostallenberg) [inline src/DependencyInjection/ConnecthollandUserExtension.php]: Doesn't this error on cache warming? As JWT_PASSPHRASE is not available as environment variable ->  https://github.com/ConnectHolland/user-bundle/pull/44/files#diff-9dcf13693e42690b6b8596e9972cd468R3

Comment (jaapromijn) [reply src/DependencyInjection/ConnecthollandUserExtension.php]: Added an empty default for now.

**Pull request merged**